engines/griffon/cutscenes.cpp

```diff
@@ -182,6 +186,20 @@ void GriffonEngine::showLogos() {
 }
 
 void GriffonEngine::intro() {
+	#ifdef USE_TTS
+		Common::TextToSpeechManager *_ttsMan = g_system->getTextToSpeechManager();
+		const char *story1 = story[0];
+		char result[1000];
+
+		strcpy(result,story1);
+
+		for(int i=10; i<=47; ++i){
```

Comment:
Using for (int i = 0; i < ARRAYSIZE(story); ++i) { would maybe be better - to not duplicate the size of the story array.

Reply:
Thank you for this feedback!  I have modified it as you said, and will proceed to fix the code as you have reviewed them.